packages/alea-frontend/components/ForumView.tsx

```diff
@@ -191,6 +191,7 @@ function ForumViewControls({
               commentId: -1,
               courseId,
               courseTerm: currentTerm,
+              institutionId: 'FAU',
```

Comment:
Create a TODO format for this project
```suggestion
              institutionId: 'FAU', // TODO: M5
```